package.json

```diff
@@ -4,27 +4,35 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node src/main/index.js",
-    "test": "./src/utils/init && jest || true && ./src/utils/destroy || true",
-    "test-on-pre-existing-cluster": "./src/utils/init-assume-cluster-exists && jest",
-    "coverage": "./src/utils/init && jest --coverage || true && ./src/utils/destroy || true",
-    "compile": "tsc",
-    "build-docker-image": "docker build . -t medicmobile/upgrade-service:0.22"
+    "start": "npx ts-node src/main/index.ts",
+    "test": "./src/utils/init && NODE_ENV=test ./node_modules/mocha/bin/mocha  --timeout 10000 --require ts-node/register ./src/__tests__/*.spec.ts || true && ./src/utils/destroy || true",
+    "test-on-pre-existing-cluster": "./src/utils/init-assume-cluster-exists && NODE_ENV=test ./node_modules/mocha/bin/mocha  --timeout 10000 --require ts-node/register ./src/__tests__/*.spec.ts",
+    "coverage": "./src/utils/init && NODE_ENV=test nyc ./node_modules/mocha/bin/mocha  --timeout 10000 --require ts-node/register ./src/__tests__/*.spec.ts || true && ./src/utils/destroy || true",
+    "build-docker-image": "docker build . -t medicmobile/upgrade-service:0.25"
```

Comment:
I think we should plan to add some e2e tests here, where we actually test the express server.